src/sdk/namespace_fs.js

```diff
@@ -346,6 +346,13 @@ class NamespaceFS {
             const file_path = this._get_file_path(params);
             const stat = await nb_native().stat(DEFAULT_FS_CONFIG, file_path);
             console.log(file_path, stat);
+            // directories are not considered objects and therefore we need
+            // to return with error not found.
+            if (stat.isDirectory()) {
```

Comment:
Not supported in the initial commit of the `naitve_fs` module.

Reply:
so for now this code will not work?

Reply:
With the current code base, it won't work.
I will now implement the option to query the directory on stat response and push it to the code base.
When it will be in the codebase I will just update you on the changes and you will push your PR as well on top of a new code base.